changelog.md

```diff
@@ -1,3 +1,43 @@
+# 3.6.0 / 2021-03-23
+
+### New Features
+
+- Advanced search available when search with Elasticsearch
+- Translation of Note and Comment directly in Tracim with Systran software (need a Systran account)
```

Comment:
Translation of Note and Comment directly in Tracim using Systran software (require a Systran account)

Reply:
requires

Reply:
Done

Reply:
it misses raphj fixes